### ISSUE TITLE
[FIX] account_payment_order: missing company currency field

### ISSUE THREAD

**Issue by micheledic** [OCA/bank-payment#1379]
## Module
account_payment_order

## Describe the bug
The field company currency field is empty on payment line

## To Reproduce
**Affected versions**:
[16.0] 
[17.0] FIX: https://github.com/OCA/bank-payment/pull/1378
1.Go to payment order
2. check payment line

**Comment by pedrobaeza** [comment]
Is v16 then affected as well?

**Comment by micheledic** [comment]
> Is v16 then affected as well?

Yes, if you show the amount_company_currency, there is not the currency symbol because the field is missing

**Comment by pedrobaeza** [comment]
Yeah, but I mean right now in the current state of the module. Can you please check and do the PR if applicable?

Thanks.
